web/router.ex

```diff
@@ -45,18 +45,18 @@ defmodule Pairmotron.Router do
   scope "/", Pairmotron do
     pipe_through [:browser, :authenticate] # Use the default browser stack
 
-    get "/pairs", PairController, :index
```

Comment:
So see https://github.com/ElixirCLE/pairmotron/issues/83, but I don't think that we should actually get rid of the `/pairs` route, I just think that it should be repurposed. I think that all of these redirects that are now redirecting to the profile are the wrong course of action. I still think that the user should see pairs in these cases, not their profile as pairs is really the central function of the app.

So I'd make the `/pairs` index just redirect to `/groups/:group_id/pairs` where the group_id at this point is just the group that the user is a member of with the lowest id (for now). In the future it'd be nice to let the user set which group they'd like to default to.

This also lowers the amount of changes you have to make in this PR (although I realize you've already made them).